### PR TITLE
Some code cleanings of problems reported by SonarQube.

### PR DESCRIPTION
**Pull request by burberius**
### PR checklist

- [x] Read the [contribution guildelines](https://github.com/swagger-api/swagger-codegen/blob/master/CONTRIBUTING.md).
- [x] Ran the shell/batch script under `./bin/` to update Petstore sample so that CIs can verify the change. (For instance, only need to run `./bin/{LANG}-petstore.sh` and `./bin/security/{LANG}-petstore.sh` if updating the {LANG} (e.g. php, ruby, python, etc) code generator or {LANG} client's mustache templates)
- [x] Filed the PR against the correct branch: master for non-breaking changes and `2.3.0` branch for breaking (non-backward compatible) changes.

### Description of the PR

SonarQube reported some bugs/problems on the library I created with swagger-codegen.
I fixed them in the template, built the codegen-cli and used that to regenerate the code of my project. The result works as expected.

### CODE REVIEW

**Comment by wing328** [comment]
@burberius thanks for the PR. Please run `./bin/java-petstore-all.sh` to update Java Petstore samples so that CIs can verify the change.

File: modules/swagger-codegen/src/main/resources/Java/libraries/jersey2/ApiClient.mustache [hunk 1]
```diff
@@ -551,13 +551,13 @@ public class ApiClient {
         filename = matcher.group(1);
     }
 
-    String prefix = null;
+    String prefix;
     String suffix = null;
```
Comment (wing328) [inline modules/swagger-codegen/src/main/resources/Java/libraries/jersey2/ApiClient.mustache]: @burberius for `suffix`, shall we remove the `null` assignment as well?

Comment (burberius) [inline modules/swagger-codegen/src/main/resources/Java/libraries/jersey2/ApiClient.mustache]: No, `suffix` is not initialized in all cases! `prefix` is!

Comment (wing328) [inline modules/swagger-codegen/src/main/resources/Java/libraries/jersey2/ApiClient.mustache]: OK. Thx for the prompt explanation.

**Comment by wing328** [comment]
@burberius  PR merged into master Thanks for your contribution.

**Pull request merged**